### PR TITLE
Cart management

### PR DESCRIPTION
**Pull request by rdthetha**
Files to check:

cartContext.js
Cart.jsx
CartProductCard.jsx

### CODE REVIEW

File: src/context/cartContext.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        <CartContext.Provider value={{state,dispatch}}>
+            {children}
+        </CartContext.Provider>
+    )
+
+
+}
+const useCart = () => useContext(CartContext);
+
+export { useCart, CartContext, CartProvider };
```
Comment (manish-GH) [inline src/context/cartContext.js:43]: There is no need to export CartContext

File: src/pages/Cart/Cart.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import React ,{useState,useEffect} from 'react'
+import { useCart } from '../../context/cartContext'
+import NavigationTop from '../../components/Navigation/navigation-top'
+import './Cart.css';
+import CartProductCard from '../../components/Card/CartProductCard'
+export default function Cart() {
+
+const {state:{cart}}=useCart();
+const [price,setPrice]=useState(0)
```
Comment (manish-GH) [inline src/pages/Cart/Cart.jsx:9]: You could consider using 1 useState instead of 3

File: src/components/Card/CartProductCard.jsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import React  from 'react'
+import { useCart } from '../../context/cartContext'
+export default function CartProductCard() {
+const {state:{cart},dispatch}=useCart();
+console.log(cart)
```
Comment (manish-GH) [inline src/components/Card/CartProductCard.jsx:5]: remove the console.log

**Review by manish-GH — APPROVED** [review]
Everything looks good. The code is easy to read and the naming convention used is good. Nice work. Just have a look at these suggestions and you are good to merge.

Suggestions: On large screens, the app looks odd. Consider centering the content
Screenshot (12)

File: src/App.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import {Route,Routes } from 'react-router-dom';
-import ProductCard from './components/Card/ProductCard';
+import Cart from './pages/Cart/Cart';
 import LandingPage from './pages/Home/LandingPage';
 import Product from './pages/Product/Product';
```
Comment (yaswanthmyneni) [inline src/App.js:5]: suggestion: create index file and export all pages from pages folder

File: src/image/images.jsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import run1 from './run-shoes.jpg'
```
Comment (yaswanthmyneni) [inline src/image/images.jsx:1]: changes: the file extension should be js instead of jsx

File: src/pages/Cart/Cart.jsx [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+<div className="ec-first-container">
+    <NavigationTop />
+    <div className="h3 center-text ec-head-text">My Cart!</div>
+    <div className="ec-cart-container">
+        <CartProductCard />
+        <div className="ec-product-total">
+            <div className="ec-bill-container">
+                <div className="h2">Price Details</div>
+                <hr />
+                <div className="ec-bill-section">
```
Comment (yaswanthmyneni) [inline src/pages/Cart/Cart.jsx:30]: suggestion: create a separate component for the bill section and call the component here

**Review by yaswanthmyneni — APPROVED** [review]
you can merge after changes

File: src/components/Card/CartProductCard.jsx (hunk 3 above)
Comment (rdthetha) [reply src/components/Card/CartProductCard.jsx:5]: Removed!

File: src/pages/Cart/Cart.jsx (hunk 6 above)
Comment (rdthetha) [reply src/pages/Cart/Cart.jsx:30]: Done!

**Pull request merged**